src/checkout/context/checkout.tsx

```diff
@@ -311,6 +315,10 @@ export const CheckoutProvider: FC<Props> = React.memo(({children}) => {
         dispatch(notify(submitError()))
       } finally {
         setIsSubmitting(false)
+        // Refresh whether user is allowed to create new orgs after upgrading to PAYG.
+        if (isFlagEnabled('createDeleteOrgs')) {
```

Comment:
what's a `delete org` and why would I create one?

smirk_emoji

Reply:
`isFlagEnabled(sassEmoji)`